### PR TITLE
Extend the datasource table

### PR DESCRIPTION
**Pull request by elemoine**
This PR replaces https://github.com/LI3DS/pg_li3ds/pull/21.

It removes the `create_project` and `delete_project` functions, meaning that the extension does no longer create project-specific schemas. Also, the datasource table is extended to be able to refer to datasets of different types.

### CODE REVIEW

File: extension/li3ds--1.0.0.sql [hunk 1]
```diff
@@ -72,7 +78,11 @@ create table session(
 
 create table datasource(
     id serial primary key
-    , uri varchar
+    -- possible uri schemes are "file" and "column"
+    , uri varchar not null constraint uri_scheme check (split_part(uri, ':', 1) = any ('{file, column}'::text[]))
+    , type datasource_type not null
+    , parameters jsonb
+    , capture_time timestamptz
```
Comment (mbredif) [inline extension/li3ds--1.0.0.sql]: This should be a time interval to accomodate for non-instantanneous datasources, (eg : lidar, route).
I would directly go to a 3D+T bbox, expressed in the datasource referential.

Comment (elemoine) [inline extension/li3ds--1.0.0.sql]: Ok. What about `bounds double precision[8]` for now?

Comment (elemoine) [inline extension/li3ds--1.0.0.sql]: Also, we already have `start_time` and `end_time` in the `session` table, which makes wonder about adding another time interval to the `datasource` table. Wouldn't that be redundant?

Comment (elemoine) [inline extension/li3ds--1.0.0.sql]: Branch updated. I replaced `capture_time` with three columns: `bounds double precision[6]`, `capture_start timestamptz` and `capture_end timestamptz`.

**Comment by elemoine** [comment]
I am going to merge this, but I am happy to change the model again if I get some feedback later after the merge.

**Pull request merged**